src/Synthesizer/Converters/Language.hs

```diff
@@ -17,17 +17,20 @@ import Synthesizer.Structure           (Channel (..), Frequency, Sample,
                                         SynSound (..))
 
 
+-- | Convert the DSL to a 'SynSound'. This allows the user to add custom sounds to the 'SynSound' if they so wish.
 convertMusicPieceToSynthesizer :: MusicPiece -> SynSound
 convertMusicPieceToSynthesizer musicPiece = SynSound $ map convertInstrumentToChannel instruments
   where
     (MusicPiece instruments) = musicPiece
 
+-- | Converts an instrument block in the DSL to a Synthesizer channel.
```

Comment:
What do you mean by instrument block?

Reply:
The `Instrument freq vol strike [ ... ]` bit of the DSL. I'll elaborate a bit

Reply:
Fixed in fcc0c66